atm.js

```diff
@@ -1,9 +1,13 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checking_balance = document.querySelector("#checking_balance");
+checkingBalance = parseInt( checking_balance.innerHTML.replace("$", "") );
```

Comment:
instead of getting the balance value from the DOM, you can create a variable to hold the balance, and update the `innerHTML` everytime the balance changes